helm-chart/mezod/values.yaml

```diff
@@ -21,8 +21,11 @@ env:
   MEZOD_CUSTOM_CONF_CONFIG_TOML: /config/config.toml.txt
   MEZOD_PORT_P2P: 26656 # Default. It should be changed in case of a custom setup
 
+secrets:
 # -- Set Secret object containing the keyring information: KEYRING_NAME, KEYRING_PASSWORD, KEYRING_MNEMONIC and ETHEREUM_ENDPOINT
-secret: "NAME_OF_THE_SECRET"
+  credentials: "YOUR_SECRET_NAME"
+# -- Set Secret object with node_key.json
+  node_key: "YOUR_SECRET_NAME"
```

Comment:
In my opinion:
1. Comments should be at the same indentation level as the code.
2. We should use `nodeKey` instead of `node_key` since we use camelCase format mostly.